Yoast/wordpress-seo
# Add config dashboard wizard notice

## Summary

This PR can be summarized in the following changelog entry:

* Showing a notice when the plugin is installed for the first time. 

## Test instructions

This PR can be tested by following these steps:
* First make sure in the option `WPSEO` the value for `show_onboarding_notice` is set to `true`.
* Also make sure there won't be a(ny) usermeta (database) value for `wpseo-dismiss-configuration-notice`
* Do a `grunt build:css` to build the CSS files.
* Go to the Yoast dashboard and see a notice.
* Open the wizard via the link in that notice
* Return to the dashboard and dismiss the notice.
* Refresh the window and see the notice won't return.
* You are done, please press merge below. Thank you.

Fixes #6932
